### PR TITLE
Add Introduction to C++ Compilers

### PR DESCRIPTION
**Pull request by jpmcb**
As an introduction to C++, this guide would be a starting point for people to learn how to run their C++ programs on the command line / terminal.

### CODE REVIEW

File: src/pages/cplusplus/Compilers.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# Intro to C++ Compilers
+
+In order to get started with C++, you will need to learn alittle about compilers and how C++ runs on your computer.
```
Comment (erictleung) [inline src/pages/cplusplus/Compilers.md]: Change "alittle" to "a little"

File: src/pages/cplusplus/Compilers.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+When all is said and done, computers only understand one language, machine language. Machine language is entirely made up of
+binary bits, or 0s and 1s. While it would be possible to program in binary, it would be incredibly tedious and time consuming. 
+So, we humans developed programming languages to make it easier to develop software. Assembly language is a direct 1 to 1 with machine
+language. Languages like C, C++, and COBOL are alittle higher and need to be compiled down. It goes even higher. Languages
```
Comment (erictleung) [inline src/pages/cplusplus/Compilers.md]: Change "alittle" to "a little"

**Comment by texas2010** [comment]
@erictleung do you wanna review it again? I fix some.

File: src/pages/cplusplus/Compilers.md [hunk 3]
```diff
@@ -23,26 +23,26 @@ is likely enough to do something on your computer, it still isn't a working exec
...
 enables you to compile your program from the command line. You can also search online for C++ compilers.
 
-2. Navigate to the source code directy
+2. Navigate to the source code directly
 
 3. Run the Compiler on your source code (assuming you are using the Microsoft Visual Studio compiler)
 `cl /EHsc helloWorld.cpp`
 
-This will now create a object file and automatically link it for you. If you look in that same folder, you will see a 
+This will now create an object file and automatically link it to you. If you look in that same folder, you will see a
```
Comment (erictleung) [inline src/pages/cplusplus/Compilers.md]: The "link it for you" was fine. So you can change it back to "...and automatically link it for you."

File: src/pages/cplusplus/Compilers.md [hunk 4]
```diff
@@ -55,7 +55,7 @@ windows may not come pre packaged with a compiler utility.
 2. Run the Compiler on your source code
 `g++ helloWorld.cpp -o helloWorld`
 
-This will create an object file and automatically link it for you. Look in the folder and you will see a helloWorld.exe 
+This will create an object file and automatically link it to you. Look in the folder and you will see a helloWorld.exe
```
Comment (erictleung) [inline src/pages/cplusplus/Compilers.md]: Same here. You can revert this back to "...and automatically link it for you."

**Comment by erictleung** [comment]
@texas2010 I've made some notes on your changes. Thanks for going through these!

**Comment by texas2010** [comment]
oh okay. I change it back.

**Pull request merged**